features/unattached_status.feature

```diff
@@ -123,3 +123,56 @@ Feature: Unattached status
            | focal   | yes      | no     | yes | yes  | yes         | yes   | no  | yes       |
            | hirsute | no       | no     | no  | no   | no          | no    | no  | no        |
            | impish  | no       | no     | no  | no   | no          | no    | no  | no        |
+
+
+    @series.all
+    @uses.config.machine_type.lxd.container
+    @uses.config.contract_token
+    Scenario Outline: Simulate status in a ubuntu machine
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I do a preflight check for `contract_token` without the all flag
+        Then stdout matches regexp:
+        """
+        SERVICE       AVAILABLE  ENTITLED   AUTO_ENABLED  DESCRIPTION
+        cc-eal        <cc-eal>    +yes  +no   +Common Criteria EAL2 Provisioning Packages
+        cis           <cis>       +yes  +no   +Center for Internet Security Audit Tools
+        esm-infra     <esm-infra> +yes  +yes  +UA Infra: Extended Security Maintenance \(ESM\)
+        fips          <fips>      +yes  +no   +NIST-certified core packages
+        fips-updates  <fips>      +yes  +no   +NIST-certified core packages with priority security updates
+        livepatch     <livepatch> +yes  +yes  +Canonical Livepatch service
+        """
+        When I do a preflight check for `contract_token` with the all flag
+        Then stdout matches regexp:
+        """
+        SERVICE       AVAILABLE  ENTITLED   AUTO_ENABLED  DESCRIPTION
+        cc-eal        <cc-eal>    +yes  +no   +Common Criteria EAL2 Provisioning Packages
+        cis           <cis>       +yes  +no   +Center for Internet Security Audit Tools
+        esm-apps      <esm-apps>  +yes  +yes  +UA Apps: Extended Security Maintenance \(ESM\)
+        esm-infra     <esm-infra> +yes  +yes  +UA Infra: Extended Security Maintenance \(ESM\)
+        fips          <fips>      +yes  +no   +NIST-certified core packages
+        fips-updates  <fips>      +yes  +no   +NIST-certified core packages with priority security updates
+        livepatch     <livepatch> +yes  +yes  +Canonical Livepatch service
+        ros           <ros>       +yes  +no   +Security Updates for the Robot Operating System
+        ros-updates   <ros>       +yes  +no   +All Updates for the Robot Operating System
+        """
+        When I do a preflight check for `contract_token` formatted as json
+        Then stdout is formatted as `json` and has keys:
+        """
+        _doc _schema_version version contract account services execution_status
+        simulated expires environment_vars
```

Comment:
Another refactor I'd like to do is to create json-schemas for all json output we have. Then we can do a deeper integration test than just checking the top-level keys.
Not necessary for this PR though

Reply:
Agree that we can add the top level keys. Although some information might be empty or not relevant, I see the concern there in the schema version.
I disagree on the `services[].auto_enabled` though, because we have no use for it in the attached status, and we already have (many) differences in the keys of a service in attached vs unattached. I think we can treat those (attached, unattached, sinulated) as different views of the services.

Reply:
Thats a really good point on auto_enabled. Let's leave it out then